src/main/java/ru/itmo/sd/deadliner2bot/service/DailyNotificationService.java

```diff
@@ -0,0 +1,24 @@
+package ru.itmo.sd.deadliner2bot.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.itmo.sd.deadliner2bot.model.DailyNotification;
+import ru.itmo.sd.deadliner2bot.repository.DailyNotificationRepository;
+
+import java.util.Set;
+
+@Service
+@RequiredArgsConstructor
+public class DailyNotificationService {
+
+    private final DailyNotificationRepository dailyNotificationRepository;
+
+    public void save(DailyNotification notification) {
+        dailyNotificationRepository.save(notification);
+    }
+
+    public Set<DailyNotification> findDailyNotificationsByChat(long chatId) {
```

Comment:
As said above, this method actually returns notifications those `notification_time` is not set yet. Please, rename the method accordingly.